_user/rotrectangle/main.c

```diff
@@ -137,67 +135,60 @@ static int rotrectangle_print(rectangle_t *self, graph_t *graphp, unsigned int a
 
 	x = x0;
 	y = y0;
-	dx = (self->a) * cos(alfa);
-	dy = (self->a) * sin(alfa);
+	dx = a * cos(alfa);
+	dy = a * sin(alfa);
 	graph_line(graphp, x, y, (int)dx, (int)dy, 1, self->color, GRAPH_QUEUE_HIGH);
 
 	x = x0 + dx;
 	y = y0 + dy;
-	dx = (self->b) * cos(alfa + (0.5 * M_PI));
-	dy = (self->b) * sin(alfa + (0.5 * M_PI));
+	dx = b * cos(alfa + (0.5 * M_PI));
+	dy = b * sin(alfa + (0.5 * M_PI));
 	graph_line(graphp, x, y, (int)dx, (int)dy, 1, self->color, GRAPH_QUEUE_HIGH);
 
 	x = x0;
 	y = y0;
-	dx = (self->b) * cos(alfa + (0.5 * M_PI));
-	dy = (self->b) * sin(alfa + (0.5 * M_PI));
+	dx = b * cos(alfa + (0.5 * M_PI));
+	dy = b * sin(alfa + (0.5 * M_PI));
 	graph_line(graphp, x, y, (int)dx, (int)dy, 1, self->color, GRAPH_QUEUE_HIGH);
 
 	x = x0 + dx;
 	y = y0 + dy;
-	dx = (self->a) * cos(alfa);
-	dy = (self->a) * sin(alfa);
+	dx = a * cos(alfa);
+	dy = a * sin(alfa);
 	graph_line(graphp, x, y, (int)dx, (int)dy, 1, self->color, GRAPH_QUEUE_HIGH);
-
-	return 0;
 }
 
 
-static int rotrectangle_save(rotrectangle_t *self)
+static void rotrectangle_save(rotrectangle_t *self)
 {
 	self->prev.xc = self->rec.xc;
 	self->prev.yc = self->rec.yc;
 	self->prev.a = self->rec.a;
 	self->prev.b = self->rec.b;
 	self->prev.color = 0;
-
-	return 0;
 }
 
 
-static int rotrectangle_rotating(rotrectangle_t *self, graph_t *graphp)
+static void rotrectangle_rotating(rotrectangle_t *self, graph_t *graphp)
```

Comment:
Now `ret` is unused. Remove it

Reply:
ret is still used in libgraph init

Reply:
GitHub Actions
/ build image (armv7a7-imx6ull)

_user/rotrectangle/main.c#L174

unused variable 'ret' [-Wunused-variable]

Reply:
https://github.com/phoenix-rtos/phoenix-rtos-project/runs/2849294798?check_suite_focus=true#step:4:660